cmd/kflex/ctx/delete_test.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2023 The KubeStellar Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package ctx
+
+import (
+	"testing"
+
+	"github.com/kubestellar/kubeflex/cmd/kflex/common"
+	"github.com/kubestellar/kubeflex/pkg/certs"
+	"github.com/kubestellar/kubeflex/pkg/kubeconfig"
+)
+
+// Test delete a context that exist and checks that it is removed
+// from the kubeconfig
+func TestDeleteOk(t *testing.T) {
+	ctxName := "cptobedeleted"
+	setupMockContext(kubeconfigPath, ctxName)
+	defer teardown(kubeconfigPath)
+
+	// Start test
+	cp := common.NewCP(kubeconfigPath, common.WithName(ctxName))
+	kconf, err := kubeconfig.LoadKubeconfig(kubeconfigPath)
+	if err != nil {
+		t.Errorf("error loading kubeconfig: %v", err)
+	}
+	err = ExecuteCtxDelete(cp, ctxName, false)
+	if err != nil {
+		t.Errorf("failed to run 'kflex ctx delete %s': %v", ctxName, err)
+	}
+	clusterName := certs.GenerateClusterName(ctxName)
+	authInfoName := certs.GenerateAuthInfoAdminName(ctxName)
+	if _, ok := kconf.Contexts[ctxName]; !ok {
+		t.Errorf("context '%s' still present in kubeconfig", ctxName)
+	}
+	if _, ok := kconf.Clusters[clusterName]; !ok {
+		t.Errorf("cluster '%s' still present in kubeconfig", clusterName)
+	}
+	if _, ok := kconf.AuthInfos[authInfoName]; !ok {
+		t.Errorf("user '%s' still present in kubeconfig", authInfoName)
```

Comment:
the messages seem wrong - this is checking that user, context etc. is NOT present, no?